mobile/src/main/java/ch/epfl/sdp/mobile/ui/prepare_game/PrepareGameScreenState.kt

```diff
@@ -0,0 +1,19 @@
+package ch.epfl.sdp.mobile.ui.prepare_game
+
+interface PrepareGameScreenState {
+  var colorChoice: ColorChoice
+  var gameType: GameType
+
+  val onNewLocalGame: () -> Unit
+  val onNewOnlineGame: () -> Unit
+}
```

Comment:
Tip : you could probably simplify your state a bit, since `gameType` is already a `var`.

```suggestion
interface PrepareGameScreenState {
  var colorChoice: ColorChoice
  var gameType: GameType
}
```